### ISSUE TITLE
Add a job button does not work in glpi 9.5.x

### ISSUE THREAD

**Issue by SUP-SIM** [fusioninventory/fusioninventory-for-glpi#3053]
Hello, I'm trying to create a job for a task, but when I click the button nothing happens.

GLPI 9.5.2 Copyright (C) 2015-2020 Teclib' and contributors
FusionInventory 9.5.0+1.0 - Copyleft © 2010-2019 by FusionInventory Team

![2020-10-23 10_59_31-GLPI - FusionInventory - 3](https://user-images.githubusercontent.com/51754048/96978698-ecbddc80-151e-11eb-8831-6b46e295567e.png)

**Comment by OliQuentDigital** [comment]
Hello, 
I have exactly the SAME ISSUE, when i want to click on the Add a job Button, nothing happen and so I am unable to create a new job for my task

**Comment by ddurieux** [comment]
Sale problem in 9.5+3.0 ?

**Comment by OliQuentDigital** [comment]
Hello, 

This is my version : 

![image](https://user-images.githubusercontent.com/35764044/114399698-821e5d00-9ba1-11eb-997d-6a074c3128b4.png)

And the Add job button is clickable but nothing happen, I click and nothing, no new screen, no error message, nothing

**Comment by ddurieux** [comment]
you have installed with marketplace?

**Comment by OliQuentDigital** [comment]
I downloaded it there : 

https://plugins.glpi-project.org/#/plugin/fusioninventory

Then I put it on my installation server via my FTP ( FileZilla ), everything else is ok and functionning but the ADD JOB BUTTON is not responding.

**Comment by ddurieux** [comment]
You got the archive fusioninventory-9.5+3.0.tar.bz2? 
If yes, open developper console in your browser (F12) and see the errors you have when you click on the button

**Comment by OliQuentDigital** [comment]
Sorry but no error messages, nothing, it seems that there is nothing " behind " the button actually.

I saw that someone else had the same error and the same problem in October 2020, and another one sooner.

**Comment by ddurieux** [comment]
I not have the problem, so I need the error to see what' is the problem.
In the developper console, can you try too but in 'network' tab?

**Comment by OliQuentDigital** [comment]
Here the screen

![image](https://user-images.githubusercontent.com/35764044/114410966-1392cc80-9bac-11eb-8813-638498572418.png)

**Comment by ddurieux** [comment]
:(
And in tab 'console' nothing?

**Comment by OliQuentDigital** [comment]
Hello, 

Here is the screen : 

![image](https://user-images.githubusercontent.com/35764044/114545440-44cad580-9c5c-11eb-8339-8e6959aef56d.png)

There is nothing at all........ and it's still non working to add a job.

**Comment by OliQuentDigital** [comment]
Can you review your code or try something else ? everything else seems to be OK

**Comment by chrhui** [comment]
Hi,
Initially wrote my 2c in #3146, but this one is closed.
I have exactly the same issue than #2837, #3146 and #3053, using an internal server running on Centos 7.6.
GLPI 9.5.6 + FusionInventory 9.5+3.0

Tried complete reinstallation of the plugin, used remi php 7.2, even 7.3, same symptoms as described in the former ticket.
Tried commenting the code as said in ticket #2837, no way.
Same using Firefox, Edge.... And in those, I see that no event is generated when clicking the button.
Did anyone of made any progress on this point?
I'm running out of ideas, please help!

**Comment by ddurieux** [comment]
Seems a problem with some installations, unable to know why, because never seen this problem :(

**Comment by chrhui** [comment]
Yes I understand, for being a dev for years, I know it is just like gambling.
Anyway, can you give any clue to track it down?
Files or functions possibly missing in glpi or FI, where the events are generated, any other button designed the same way that I can examine for comparision...
N'importe quoi qui puisse me donner des pistes en somme.

**Comment by ddurieux** [comment]
The best solution is for me is to check directly on your glpi / server

**Comment by chrhui** [comment]
That's what I did. None of the log is reporting anything, exactly as @OliQuentDigital reported, Just as if I didn't clicked on the button.
Y'a t'il d'autres boutons dans FI qui soient fait de la meme maniere?

**Comment by ddurieux** [comment]
It's in javascript, so 2 possibilities : 
1/ error with javascript
2/ error with the webpage requested by the javascript

**Comment by ascrypt** [comment]
I even cannot see Add a job button. 9.5.6 +3.0

**Comment by ddurieux** [comment]
Have you errors in the console of your browser (F12)? in console and in network

**Comment by ascrypt** [comment]
@ddurieux No error in browser console, php_error and apache error logs

**Comment by ddurieux** [comment]
Can you 'inspect' the button and do a screenshot of the `<form>` code, like in my screenshot?

![image](https://user-images.githubusercontent.com/239744/148675463-4d16b86b-be8d-4ee1-b4a5-a2f7cd55cc85.png)

**Comment by ascrypt** [comment]
![kkk](https://user-images.githubusercontent.com/16368249/149620244-fd240265-ec86-43e4-9af2-25e1bdf6da32.PNG)

**Comment by ddurieux** [comment]
Seems right... Is your GLPI is available on the internet? if yes perhaps you can give me access to it to try to see the problem.

**Comment by ascrypt** [comment]
@ddurieux The problem was related to my CSS style customization. I fixed it and also button works correctly. Thanks for your help

**Comment by ddurieux** [comment]
You CSS style customization? can you explain more for other users have the same problem?

Thanks ;)

**Comment by ascrypt** [comment]
I added this style incorrectly : `.tab_cadrehov tbody tr:last-child,.tab_glpi:last-of-type{display:none}`

**Comment by ddurieux** [comment]
ok thanks ;)

**Comment by perrinegarnier13** [comment]
Hello,

I have the same issue, when i click on add for a new config job for configure the discovery network nothing is happening. 
I am on [GLPI 9.5.7] Windows server 2019 (IIS), PHP 7.5.27, Mysql 8.0 ; I try [FusionInventory 9.5.0+1.0]2.0]3.0] same issue.
My GLPI is not connected to internet and i download the plugin on github.

I have nothing in the console when i click on "Add a job":
![image](https://user-images.githubusercontent.com/100209396/155139042-5912a5b3-6249-4b1b-87e4-81c1520373c4.png)

Maybe It is a javascript problem, when i am on the menu.php. I have this issue already: 
![image](https://user-images.githubusercontent.com/100209396/155138742-db5cc867-da2e-4a9d-a170-2591a909d267.png)

thanks in advance for your help.
Perrine

**Comment by perrinegarnier13** [comment]
Bonjour @ddurieux ,

J'ai l'impression que le problème vient du fait que nos GLPI ne sont pas connectés à internet du coup le jquery ne se charge pas depuis le webpack. Est-ce qu'il y aurait un moyen de mettre ces sources en local sur notre serveur et de dire à votre plugin de se baser sur ce dossier ? Car ma machine est isolé niveau réseau et elle ne peux donc pas sortir sur le web.

Merci pour ton retour

**Comment by perrinegarnier13** [comment]
@ddurieux

**Comment by ddurieux** [comment]
Je veux bien mais normalement tout est en local, tu as la liste des requêtes qui essayent de partir sur le net quand tu charge la page?

**Comment by sygfi** [comment]
Hi @ddurieux,
I have same problem with GLPI10.0.7, when I click on Add a Job nothing happen.
Do you have some news about this issue ?

**Comment by sygfi** [comment]
Does anyone find a solution or a workaround ?
